languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -19345,6 +19345,542 @@ USA
     <!-- Grammar -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammar" type="grammar">
+        <rulegroup id="ITS_TO_IT_S" name="I have to do laundry while its (it's) still sunny" default="temp_off">
+            <rule>
+                <antipattern>
+                    <token>its</token>
+                    <token postag="NN">
+                        <exception>like</exception>
+                    </token>
+                </antipattern>
+                <antipattern>
+                    <token>its</token>
+                    <token postag="JJ">
+                        <exception>like</exception>
+                    </token>
+                    <token postag="NN" />
+                </antipattern>
+                <antipattern>
+                    <token>its</token>
+                    <token regexp="yes">are|did|seemed|was</token>
+                </antipattern>
+                <antipattern>
+                    <token>its</token>
+                    <token regexp="yes">re.*
+                        <exception regexp="yes">.*(ing|ed)</exception>
+                    </token>
+                </antipattern>
+                <antipattern>
+                    <token>due</token>
+                    <token>to</token>
+                    <token>its</token>
+                </antipattern>
+                <pattern>
+                    <marker>
+                        <token regexp="yes" case_sensitive="yes">Its|its</token>
+                    </marker>
+                    <token chunk="B-VP">
+                        <exception postag="MD|NNS|NN:UN?|VBZ" postag_regexp="yes" />
+                    </token>
+                </pattern>
+                <message>Did you mean it's?</message>
```

Comment:
`<message>Did you mean "it's"?</message>`

Reply:
yeah lemme fix that...